### DISCUSSION TITLE
VEC Pack 1) Space Update

### DISCUSSION THREAD

**Discussion by projectkepler-ru** [Bubberstation/Bubberstation#1930] [codebase discussion]
This is the most voted option for the first half, with 5 additional planned PR afterward.

### Important

What this is about:
Space Exploration has been a long desired aspect of SS13
The result has been that Ghost Role, Space Ruins (primarily lore and story oriented one) is highly desirable. Henceforth, we will focus on that
Comment below for discussion

**Comment by ReturnToZender** [comment]
The expeditions

**Reply by projectkepler-ru** [reply]
Yes!

**Comment by AllCorruptingWyrm** [comment]
So I think we should add a role called "Marine" and let them board ships/ruins.

**Reply by projectkepler-ru** [reply]
Planned ish as a subtype of miner

**Comment by Erol509** [comment]
Overmap :)
